cmd/bee/cmd/start.go

```diff
@@ -404,3 +419,33 @@ func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (co
 		pssPrivateKey:    pssPrivateKey,
 	}, nil
 }
+
+func parseNetworks(main, test bool, networkID uint64) (uint64, error) {
+	switch {
+	case main && test:
+		return 0, errors.New("please provide either mainnet or testnet option")
+	case main && networkID != 1:
+		return 0, errors.New("provided network ID does not match mainnet")
+	case test && networkID != 10:
+		return 0, errors.New("provided network ID does not match testnet")
+	case main:
+		return 1, nil
+	case test:
+		return 10, nil
+	default:
+		return networkID, nil
+	}
+}
+
+func parseBootnodes(main, test bool, bootnodes []string) ([]string, error) {
+	switch {
+	case len(bootnodes) > 0: // use the user provided values
+		return bootnodes, nil
+	case main:
+		return []string{"/dnsaddr/mainnet.ethswarm.org"}, nil
+	case test:
+		return []string{"/dnsaddr/testnet.ethswarm.org"}, nil
+	default:
+		return nil, errors.New("bootnode required when not connecting to the official main or test networks")
```

Comment:
why?

Reply:
standalone nodes do not require bootnodes, I believe